portingdb/templates/graph.html

```diff
@@ -47,31 +47,13 @@
     <div class="col-12">
         <h1>A Graph</h1>
         <p>
-            Here is a graph showing all dependency relationships for unported
-            packages (idle, blocked, and mispackaged)
-            {% if grp %}
-                in the <a href='{{ url_for("group", grp=grp) }}'>{{ grp }}</a>
-                group
-            {% elif pkg %}
-                related to the
-                <a href='{{ url_for("package", pkg=pkg) }}'>{{ pkg }}</a>
-                package, as
-            {% endif %}
-            tracked by <a href='{{ url_for("hello") }}'>portingdb</a>.
-            Packages with no relationship to other unported packages are not
-            shown.
-            Darker colors mean more dependencies.
+            Here is a graph showing dependency relationships for
+            packages in Fedora Rawhide that need Python 2 in some way.
+            Brighter colors roughly mean the package can be removed sooner.
```

Comment:
OK, I guess I don't get that part. I see the same colors as in portingdb. How is their brightness showing when they can be removed?

Reply:
The ones that can't be removed easily are dark (and they're in the center).
Granted, it's not very obvious – and perhaps not even helpful. Should this just use the same colors as everywhere else?
Here's the graph with this shading turned off:

![btightgraph](https://user-images.githubusercontent.com/302922/49594215-ab95a100-f975-11e8-8ab0-b60d92cc090e.png)

Reply:
I only see it on the green ones. And only now when you showed me. Better ask another person as well.

Reply:
Well, the "hard to remove" ones *are* mostly green (and gray).

Reply:
Are we bikeshedding at this point?

Reply:
Yes. Keep the colors.